eox_hooks/actions.py

```diff
@@ -0,0 +1,112 @@
+"""
+Module for actions.
+"""
+import logging
+
+import requests
+
+from eox_hooks.serializers import CertificateSerializer, CourseSerializer, UserSerializer
+from eox_hooks.utils import _get_course, flatten_dict, get_action_settings
+
+log = logging.getLogger(__name__)
+
+
+def post_to_webhook_url(**kwargs):
```

Comment:
I think that we need to find a way to use a namespace, so we can define different fields for different hooks. Probably there is already an argument here that we can use for that. What do you think? @magajh

Reply:
@morenol  I agree. I changed it so now the trigger event is passed to the action in the kwargs. It also changed the way the settings are defined, I updated the PR description and explained how the settings should be passed now.